node/src/accountant/scanners/mod.rs

```diff
@@ -1380,11 +1384,14 @@ mod tests {
     fn payable_scanner_throws_error_in_case_no_qualified_payable_is_found() {
         let now = SystemTime::now();
         let (_, unqualified_payable_accounts, _) =
-            make_unqualified_and_qualified_payables(now, &PaymentThresholds::default());
+            make_qualified_and_unqualified_payables(now, &PaymentThresholds::default());
         let payable_dao =
             PayableDaoMock::new().non_pending_payables_result(unqualified_payable_accounts);
         let mut subject = PayableScannerBuilder::new()
             .payable_dao(payable_dao)
+            .payable_inspector(PayableInspector::new(Box::new(
+                PayableThresholdsGaugeReal::default(),
+            )))
```

Comment:
It seems that there is a wide usage of the code below.

```rust
            .payable_inspector(PayableInspector::new(Box::new(
                PayableThresholdsGaugeReal::default(),
            )))
```

If we could make it part of the `default()`, or `new()` in this case of the `PayableScannerBuilder()`, it would eliminate the need of supplying in the tests.

Optional: Since, you might also like to see the usage of `default()` instead of `new()`, when there aren't any arguments, you may change it only for this structure. We may plan to do it for others too but in a new card.